be/src/exec/vectorized/olap_scan_node.cpp

```diff
@@ -152,12 +155,12 @@ Status OlapScanNode::get_next(RuntimeState* state, ChunkPtr* chunk, bool* eos) {
         *eos = false;
         DCHECK_CHUNK(*chunk);
         return Status::OK();
-    } else {
-        _update_status(Status::EndOfFile("EOF of OlapScanNode"));
-        *eos = true;
-        status = _get_status();
-        return status.is_end_of_file() ? Status::OK() : status;
     }
+
```

Comment:
why change this？

Reply:
No specific reason. I prefer this style and it's the same code pattern in hdfs_scan_node.cpp.